src/main/java/uk/gov/companieshouse/itemhandler/kafka/OrdersKafkaConsumer.java

```diff
@@ -122,6 +126,8 @@ protected void handleMessage(org.springframework.messaging.Message<OrderReceived
             logMessageReceived(message);
 
             // process message
+            // TODO GCI-931 Exceptions?
```

Comment:
I can see that this TODO references this PR's story - will this need to be done before approval?

Reply:
Good spot! No, sorry, will not be resolving this here. I will remove this TODO.